tutorial/sample-problems/VectorCalc/VectorLineSegment2.pg

```diff
@@ -41,14 +42,14 @@ $ans = Compute("$P + $speed * t * $disp / norm($disp)");
 
 #:% section = statement
 BEGIN_PGML
-A particle starts at the point [`P = [$P]`] when [`t = 0`] and moves along a
-straight line toward [`Q = [@ Point($P + $disp) @]`] at a speed of [`[$speed]`]
-centimeters per second.  Assume that [`x`], [`y`], and [`z`] are measured in
-centimeters.
+A particle starts at the point [`P = [$P]`] when [`t = 0`]
+and moves along a straight line toward [`Q = [@ Point($P + $disp) @]`]
+at a speed of [`[$speed]`] centimeters per second . Assume that [`x`],
+[`y`], and [`z`] are measured in centimeters.
```

Comment:
Put this back to the way it was.  This is unnecessary reformatting, and isn't even formatted as well as it was.